src/main/java/fruitshop/strategy/OperationStrategy.java

```diff
@@ -0,0 +1,8 @@
+package fruitshop.strategy;
+
+import fruitshop.model.FruitTransaction;
+
+public interface OperationStrategy {
+    public void getOperation(FruitTransaction fruitTransaction);
```

Comment:
The method name `getOperation` implies that it should return an `OperationHandler` instance, but it has a `void` return type. This could be misleading. Consider renaming the method to reflect the action it performs, such as `executeOperation` or `apply`, and ensure it returns an `OperationHandler` if that was the original intent.